develop/TOOLS/ThirdPartyTools/DataStax_DevCenter.md

```diff
@@ -4,16 +4,49 @@ uid: DataStax_DevCenter
 
 # DataStax DevCenter
 
-This tool allows you to perform CQL queries or run scripts on a Cassandra database.
+> [!NOTE]
+> The recommended setup for DataMiner storage is [Storage as a Service](xref:STaaS).
+
+This tool allows you to perform CQL queries or run scripts on a Cassandra database. It can be used on any DMA using a Cassandra local database.
```

Comment:
Does "on any DMA Cassandra local database" mean that this can only be used if Cassandra has been installed on the same machine as the DMA?

Reply:
Good eye, looks like I didn't adjust this sentence completely.
"The tool can be used on any DMA using a Cassandra local database."
should actually be
"The tool can be used on any server using a Cassandra database."